c/index.md

```diff
@@ -0,0 +1,10 @@
+# List of artists starting with C
+
+- [Claudiot](claudiot.md)
+- [Claud de Mahanoro](claud-de-mahanoro.md)
+- [Clo mahajanga](clo-mahajanga.md)
+- [Clauvis](clauvis.md)
+- [Chantal](chantal.md)
```

Comment:
Please arrange this list alphabetically.

Reply:
Hey, 
Here it is, please check.